lib/rspec/rails/fixture_support.rb

```diff
@@ -12,8 +12,14 @@ module FixtureSupport
         # @private prevent ActiveSupport::TestFixtures to start a DB transaction.
         # Monkey patched to avoid collisions with 'let(:name)' in Rails 6.1 and after
         # and let(:method_name) before Rails 6.1.
-        def run_in_transaction?
-          use_transactional_tests && !self.class.uses_transaction?(self)
+        if ::Rails::VERSION::STRING > '5'
```

Comment:
This check is inspired by 

https://github.com/rspec/rspec-rails/blob/c100cf1c26bc4f20f54900f22b360b6c2d0553a8/lib/rspec/rails/fixture_support.rb#L24